_posts/2018-10-04-all-docker.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Docktoberfest"
+date: 2018-10-04 09:30
+categories: upcoming
+---
+
+- Location: [Walter Library](http://campusmaps.umn.edu/walter-library), Room 402
+- Day: Thursday, October 4
+- Time: 9:30 - 11:00
+
+Agenda:
+
+- *9:30 - 9:50* **What is Docker and "Why Should I Care?"**, Andrew Zenk
+- *9:50 - 10:20* **Docker in Development with Docker Compose**, Ian Whitney and Remy Abduallhi
```

Comment:
I think this is just Remy, I am checking

Reply:
We're actually figuring that out today. Should have an update by the end of the workday.

Reply:
Got confirmation that it's just Remy.